iot_amqp_oca/tests/__init__.py

```diff
@@ -1,4 +1 @@
 from . import test_amqp
-from odoo.addons.iot_output.tests import test_iot
```

Comment:
Why did you remove that?

That made some sense, but maybe you should need to do 
`from odoo.addons.iot_output import tests`